action.yml

```diff
@@ -25,6 +25,12 @@ inputs:
     default: "broad-jade-dev"
   k8_cluster:
     description: "Google kubernetes cluster for sdk ie:integration-master"
+  google_application_credentials:
+    description: "Path where google credential json is stored"
+    default: "/tmp/jade-dev-account.json"
+  google_application_credentials_pem:
+    description: "Path where google credential pem is stored"
+    default: "/tmp/jade-dev-account.pem"
```

Comment:
Do we still need the PEM file? I think it's no longer a part of the SA specification.

Reply:
Ok, I ran a test and the integration tests are definitely still expecting this pem file. Example error [scan here](https://scans.gradle.com/s/gauwqowqgbgzs/tests/:testIntegration/bio.terra.service.snapshot.SnapshotTest/snapshotUnauthorizedPermissionsTest#1).
So, any of my changes here are for the shared code and should just be used for the integration tests.

Reply:
Right, the integration tests definitely still need the PEM file. That has not changed. But I'm confused why you need to add this if the test runner isn't using it. Is this just general cleanup?

Reply:
Should be removed now!